Exiled.Events/Patches/Events/Server/RespawningTeam.cs

```diff
@@ -46,16 +46,9 @@ private static bool Prefix(RespawnManager __instance)
                             .Where(p => p.IsDead && !p.IsOverwatchEnabled));
 
                     if (__instance._prioritySpawn)
-                    {
-                        var tempList = ListPool<API.Features.Player>.Shared.Rent();
-                        tempList.AddRange(list.OrderBy(item => item.ReferenceHub.characterClassManager.DeathTime));
-                        ListPool<API.Features.Player>.Shared.Return(list);
-                        list = tempList;
-                    }
+                        list.OrderBy(item => item.ReferenceHub.characterClassManager.DeathTime).ToList();
```

Comment:
as far as I can tell, ToList() creates a new list and that list isn't assigned to the `list` variable

Reply:
Fixed